### DISCUSSION TITLE
PropertyPanel getTemplate from a HTML file?

### DISCUSSION THREAD

**Discussion by MxRaccoon** [serenity-is/Serenity#7233] [General]
Hi is there a way to use / call a html file instead of using the getTemplate() in a PropertyPanel page like in the TemplatedWidget?

thanks in advance.

**Comment by MxRaccoon** [comment]
I am going to update my question as I am now a little more aware of how this works. I am building some templeted panel pages for the dashboard and for a mini site out the user login (now on external mini site), at first I build the dashboard using a TemplatedWidget and didnt used the getTemplate() funtion, instead I put an html file on the same folder named "DashboardPage.Template", everything works fine on debug but once I published the site throws an error "Uncaught Error: Can't load script data: Template.DashboardPage!" So I used the getTemplate() and the httml inside the told file and it works now on the published site. So the question is if there is some way to implement the html file instead the function? on the external site I have large html code and it is easy to read and work with when is in the file.

Hope some one has some Idea, I tried already to read the file an return the html as string on the getTemplate() function but didnt work.

thanks in advance

**Comment by VictorTomaili** [comment]
html templates are deprecated, use tsx files instead. Currently it is continuing to work for backward compatibility.
